Project.toml

```
@@ -0,0 +1,19 @@
+name = "IndirectArrays"
+uuid = "9b13fd28-a010-5f03-acff-a1bbcff69959"
+
+[deps]
+Compat = "34da2185-b29b-5c13-b0c7-acf172513d20"
+
+[compat]
+Compat = "0.53"
+FixedPointNumbers = "0.3"
```

```suggestion
FixedPointNumbers = "0.6.1, 0.7"
```
0.3 doesn't work on Julia 1.x, and your version blocks it from using any newer version.